src/controllers/collisions.rs

```diff
@@ -7,15 +11,26 @@ const SCORE_PER_ENEMY: u32 = 10;
 pub struct CollisionsController;
 
 impl CollisionsController {
-    pub fn handle_collisions(state: &mut GameState) {
-        CollisionsController::handle_bullet_collisions(state);
-        CollisionsController::handle_player_collisions(state);
+    pub fn handle_collisions(app: &mut ApplicationState, ctx: &mut Context) {
+        // Don't handle collisions if the player is dead
+        if app.game_state.world.player.is_dead { return; }
```

Comment:
What happens when there are still bullets flying around? It would be a nice effect to allow them to still kill enemies (though this probably won't happen very often)

Reply:
:+1: